CoreFoundation/String.subproj/CFCharacterSet.c

```diff
@@ -1365,7 +1371,9 @@ Boolean _CFCharacterSetInitWithCharactersInString(CFMutableCharacterSetRef cset,
         
         if (0 == length) {
             __CFCSetPutHasHashValue(cset, true); // _hashValue is 0
-        } else if (length > 1) { // Check for surrogate
```

Comment:
Please don't move the comments as it increases noise in the delta. Can you move that back here, and then it will be clearly seen you're just adding a new line in.

Reply:
Yes, this comment is not related to the line that I was adding, so I decided to move it below, if you think it's still will be clear where it will "Check for surrogate" - I can leave it where it was.